### DISCUSSION TITLE
healer nupkg consumptions

### DISCUSSION THREAD

**Discussion by kumarnareshh74** [microsoft/service-fabric-healer#164] [Q&A]
1. how can we write custom predicates/repairs when consume healer as a nuget package?
2. Also, is there a way to hook into during service start - like the customobserverstartup code in fabric observer - https://github.com/microsoft/service-fabric-observer/blob/main/SampleObserverPlugin/SampleNewObserverStartup.cs

**Comment by GitTorre** [comment]
Thank you for the suggestion. I have created Issue #165 for tracking.
